tests/python-pytest/onnx/export/mxnet_export_test.py

```diff
@@ -214,6 +214,80 @@ def test_spacetodepth():
 
     npt.assert_almost_equal(output[0], numpy_op)
 
+@with_seed()
+def test_logical_and():
+    """Test for logical and in onnx operators."""
```

Comment:
Can we have common method get_input() for getting input arrays which we can use in all functions?

Reply:
@Roshrini as discussed offline, added a helper function to get integer inputs